ctrd/container.go

```diff
@@ -190,7 +190,7 @@ func (c *Client) DestroyContainer(ctx context.Context, id string) (*Message, err
 	}
 
 	waitExit := func() *Message {
-		return c.ProbeContainer(ctx, id, time.Second*5)
+		return c.ProbeContainer(ctx, id, time.Duration(timeout)*time.Second)
```

Comment:
We can directly use `timeout * time.Second`, right?

Reply:
right

Reply:
Do we have a plan to change this? 😄

Reply:
it seems that we must use `time.Duration()` when we use variable，we can use this way if we use the constant.